src/main/java/pages/ElectronicsPage.java

```diff
@@ -13,11 +13,13 @@ public ElectronicsPage() {
         super();
     }
 
-    public void clickOnMobilePhoneLink() {
+    public ElectronicsPage clickOnMobilePhoneLink() {
         mobilePhoneLink.click();
+        return new ElectronicsPage();
```

Comment:
How can you replace new ElectronicsPage(); here? As you are referring to it inside this class itself?